### PR TITLE
Web api of stock source links management

### PR DESCRIPTION
**Pull request by naydav**
https://github.com/magento-engcom/magento2/issues/83

- Need to describe web api for stock source links management
- Cover with api functional tests

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-engcom/magento2?pullRequest=84) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-engcom/magento2?pullRequest=84) before we can accept your contribution.<br/>

File: app/code/Magento/InventoryApi/etc/acl.xml [hunk 1]
```diff
@@ -14,7 +14,8 @@
                 </resource>
                 <resource id="Magento_InventoryApi::stock" title="Inventory Stocks" translate="title" sortOrder="20">
                     <resource id="Magento_InventoryApi::stock_edit" title="Edit Inventory Stocks" translate="title" sortOrder="10" />
-                    <resource id="Magento_InventoryApi::source_delete" title="Delete Inventory Stocks" translate="title" sortOrder="20" />
+                    <resource id="Magento_InventoryApi::stock_delete" title="Delete Inventory Stocks" translate="title" sortOrder="20" />
```
Comment (maghamed) [inline app/code/Magento/InventoryApi/etc/acl.xml:17]: why do we introduce independent resource for Stock Edit and Stock Delete? 
why not to have the single ACL resource for both operations? 

Like this one:
```xml
 <resource id="Magento_InventoryApi::stock" title="Manage Inventory Stocks" translate="title" sortOrder="20" />
```

And the same for Source

**Review by maghamed — APPROVED** [review]
Everything except ACL looks good for me

File: app/code/Magento/InventoryApi/etc/acl.xml (hunk 1 above)
Comment (naydav) [reply app/code/Magento/InventoryApi/etc/acl.xml:17]: Separate ticket has been created
https://github.com/magento-engcom/magento2/issues/89
need to discuss several things in a scope of ACL refactoring

**Comment by naydav** [comment]
Internal buid https://bamboo.corp.magento.com/browse/M2-AT2506-2

**Comment by larsroettig** [comment]
@naydav https://bamboo.corp.magento.com/ is magento internal domain :(

**Pull request merged**